test/vshard_helpers/instances/storage.lua

```diff
@@ -0,0 +1,35 @@
+local fio = require('fio')
+
+local appdir = fio.abspath(debug.sourcedir() .. '/../../../')
+if package.setsearchroot ~= nil then
+    package.setsearchroot(appdir)
+else
+    package.path = package.path .. appdir .. '/?.lua;'
+    package.path = package.path .. appdir .. '/?/init.lua;'
+    package.path = package.path .. appdir .. '/.rocks/share/tarantool/?.lua;'
+    package.path = package.path .. appdir .. '/.rocks/share/tarantool/?/init.lua;'
+    package.cpath = package.cpath .. appdir .. '/?.so;'
+    package.cpath = package.cpath .. appdir .. '/?.dylib;'
+    package.cpath = package.cpath .. appdir .. '/.rocks/lib/tarantool/?.so;'
+    package.cpath = package.cpath .. appdir .. '/.rocks/lib/tarantool/?.dylib;'
+end
+
+local utils = require('test.vshard_helpers.instances.utils')
+
+-- Do not load entire vshard into the global namespace to catch errors when
+-- code relies on that.
+_G.vshard = {
+    storage = require('vshard.storage'),
+}
+_G.ivshard = _G.vshard
```

Comment:
What is ivshard?

Reply:
An artifact of the previous implementation. Removed.